index.html

```diff
@@ -1,16 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
+  <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vanilla Todo</title>
     <link rel="stylesheet" href="style.css" />
-</head>
-
-<body>
-    <div class="container"></div>
-</body>
-<script src="script.js"></script>
+  </head>
 
-</html>
\ No newline at end of file
+  <body>
+    <div class="container">
```

Comment:
list의 개수가 많아질 경우 처음 의도한 영역을 벗어나는 오류가 있습니다. 
css로 원하는 영역에만 표시되도록 한정지어줄 필요가 있을 것 같습니다!

css에 다음과 같이 추가하면 됩니다!
```suggestion
.pending-list,
.done-list{
flex-grow: 1;
overflow-y: scroll;
}
```